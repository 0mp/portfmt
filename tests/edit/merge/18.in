USE_SDL=	sdl2
USE_CXXSTD=	c++98
