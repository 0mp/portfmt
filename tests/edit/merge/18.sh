${PORTEDIT} merge -e 'LDFLAGS_i386=-Wl,-znotext' 18.in | diff -L 18.expected -L 18.actual -u 18.expected -
