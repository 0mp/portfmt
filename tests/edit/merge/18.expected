USE_SDL=	sdl2
USE_CXXSTD=	c++98

LDFLAGS_i386=	-Wl,-znotext
